ansys/mapdl/core/post.py

```diff
@@ -397,6 +881,51 @@ def _plot_point_scalars(self, scalars, show_node_numbering=False, **kwargs):
 
         return general_plotter(meshes, [], labels, **kwargs)
 
+    def _plot_cell_scalars(self, scalars, show_elem_numbering=False, **kwargs):
+        """Plot cell scalars.
+
+        Assumes scalars are from all elements and not just the active surface.
+        """
+        if not scalars.size:
+            raise RuntimeError(
+                "Result unavailable.  Either the result has not been loaded "
+                "with ``mapdl.set(step, sub_step)`` or the result does not "
+                "exist within the result file."
+            )
+
+        surf = self._mapdl.mesh._surf
+
+        # as ``disp`` returns the result for all nodes, we need all node numbers
+        # and to index to the output node numbers
+        if hasattr(self._mapdl.mesh, "enum_all"):
+            enum = self._mapdl.mesh.enum
+        else:
+            enum = self._all_enum
+
+        # it's possible that there are duplicated element numbers,
+        # therefore we need to get the unique values and a reverse index
+        uni, ridx = np.unique(surf["ansys_elem_num"], return_inverse=True)
+        mask = np.in1d(enum, uni, assume_unique=True)
```

Comment:
Numpy documentation recommends to use ``isin`` instead of ``in1d``. 

```suggestion
        mask = np.isin(enum, uni, assume_unique=True)
```

Reply:
https://numpy.org/doc/stable/reference/generated/numpy.in1d.html

Reply:
Good catch.